tests/Element/DocumentElementTestCaseTest.php

```diff
@@ -4,7 +4,7 @@
 
 use Behat\Mink\Element\DocumentElement;
 
-class DocumentElementTest extends ElementTest
+class DocumentElementTestCaseTest extends ElementTestCase
```

Comment:
This should still be named `DocumentElementTest`. They are the tests for `DocumentElement`